packages/headless/src/features/facet-options/facet-options-selectors.ts

```diff
@@ -6,3 +6,15 @@ export const isFacetEnabledSelector = (
 ) => {
   return state.facetOptions.facets[id]?.enabled ?? true;
 };
+
+export const selectFacetOptions = (state: {
+  facetOptions?: {freezeFacetOrder?: boolean};
+}) => {
+  if (state.facetOptions?.freezeFacetOrder === undefined) {
+    return undefined;
+  }
+
+  return {
+    freezeFacetOrder: state.facetOptions.freezeFacetOrder,
+  };
+};
```

Comment:
here you could do the following instead (just a suggestion)

```suggestion
export const selectFacetOptions = (state: {
  facetOptions?: { freezeFacetOrder?: boolean };
}) => {
  const { freezeFacetOrder } = state.facetOptions ?? {};
  return freezeFacetOrder !== undefined ? { freezeFacetOrder } : undefined;
};
```